tests/ga/test_update.py

```diff
@@ -2711,10 +2711,8 @@ def create_log_and_telemetry_mocks():
             with create_log_and_telemetry_mocks() as (logger, add_event):
                 update_handler._try_update_goal_state(protocol)
 
-                w = warnings()
-                pw = periodic_warnings()
-                self.assertEqual(1, len(w), "A failure should have produced a warning: [{0}]".format(w))
-                self.assertEqual(1, len(pw), "A failure should have produced a periodic warning: [{0}]".format(pw))
+                e = errors()
+                self.assertEqual(1, len(e), "A failure should have produced a warning: [{0}]".format(e))
```

Comment:
NIT: more accurately, "a failure should have produced an error".

Reply:
thanks, fixed